cli/tests/pcluster/validators/test_all_validators/test_all_validators_are_called/slurm_1.yaml

```diff
@@ -139,7 +139,7 @@ SharedStorage:
       # BackupId: backup-fedcba98 # BackupId cannot coexist with some of the fields
       KmsKeyId: String  # xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
       # FileSystemId: fs-12345678123456789 # FileSystemId cannot coexist with some of the fields
-      AutoImportPolicy: NEW  # NEW | NEW_CHANGED
+      AutoImportPolicy: NEW  # NEW | NEW_CHANGED | NEW_CHANGED_DELETED
```

Comment:
The AutoImport policy seems a test parameter according to `tests/integration-tests/tests/storage/test_fsx_lustre.py`.

So, why not using Jinja variables here instead of hard wired value.

Reply:
This is a unit test for validators, not related to test_fsx_lustre integration tests.